src/setup_node_env/babel_register/register.js

```diff
@@ -38,12 +38,12 @@ var ignore = [
   // https://github.com/elastic/kibana/issues/14800#issuecomment-366130268
 
   // ignore paths matching `/node_modules/{a}/{b}`, unless `a`
-  // is `x-pack` or `b` is not `node_modules`
+  // is `x-pack` and `b` is not `node_modules`
   /\/node_modules\/(?!x-pack\/(?!node_modules)([^\/]+))([^\/]+\/[^\/]+)/,
 
   // ignore paths matching `/canvas/canvas_plugin/{a}/{b}` unless
   // is `x-pack` and `b` is not `node_modules`
-  /\/canvas\/canvas_plugin\/(?!functions\/server)([^\/]+\/[^\/]+)/,
+  /\/node_modules\/(?!(x-pack\/|@kbn\/interpreter\/)(?!node_modules)([^\/]+))([^\/]+\/[^\/]+)/,
```

Comment:
We should change this again to `/\/canvas\/canvas_plugin\/(?!functions\/server)([^\/]+\/[^\/]+)/,`